### PR TITLE
Maya: Options for VrayProxy output - OP-2010

### PR DESCRIPTION
**Pull request by tokejepsen**
## Brief description
Options for output of VrayProxy.

## Description
Client requested more granular control of output from VrayProxy instance. Exposed options on the instance and settings for vrmesh and alembic.

## Testing notes:
1. Create VrayProxy in Maya.
2. Enable/Disable attributes `vrmesh` and `alembic`, on the instance.
3. Enable/Disable `project_settings/maya/create/CreateVrayProxy/vrmesh` and/or `project_settings/maya/create/CreateVrayProxy/alembic` and create VrayProxy instance to validate settings are propagated to instance.

### CODE REVIEW

**Review by LiborBatek — APPROVED** [review]
Did tested and all works as expected. 
After creating ```vrayproxy``` instance there are both ```alembic``` and ```vrmesh``` available.

image

When published both representations are available to load (alembic/vrmesh) and correctly loads via ```Asset Loader```

image

File: openpype/hosts/maya/plugins/publish/validate_vrayproxy.py [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-        invalid = self.get_invalid(instance)
-        if invalid:
-            raise RuntimeError("'%s' has invalid settings for VRay Proxy "
-                               "export!" % instance.name)
-
-    @classmethod
-    def get_invalid(cls, instance):
         data = instance.data
+        self.log.info(data["family"])
+        self.log.info(data["families"])
```
Comment (moonyuet) [inline openpype/hosts/maya/plugins/publish/validate_vrayproxy.py:16]: Not sure if it is neccessary to include the logging information....

Comment (tokejepsen) [inline openpype/hosts/maya/plugins/publish/validate_vrayproxy.py:16]:
```suggestion
```

Comment (tokejepsen) [reply openpype/hosts/maya/plugins/publish/validate_vrayproxy.py]: Good point! Its gone.

**Comment by tokejepsen** [comment]
@moonyuet you happy with the changes?

**Review by moonyuet — APPROVED** [review]

**Review by antirotor — APPROVED** [review]

**Pull request merged**